rollup.config.js

```diff
@@ -0,0 +1,7 @@
+export default {
+    input: 'lib/index.js',
+    output: {
+      file: 'dist/index.js',
+      format: 'cjs',
+    }
+  };
```

Comment:
Please always run `npm lint` or `npm lint:fix` in order to ensure all code is complying with [standard format](https://standardjs.com/)

Reply:
Sorry, I just realized, that the lint command doesn't cover files beyond the `lib` folder. Please run `npm standard --fix ./rollup.config`, I will update the lint scope in an own PR, since this also affects the CI